frontend/website/pages/docs/my-first-transaction.mdx

```diff
@@ -8,13 +8,12 @@ In this section, we'll make our first transaction on the Coda network. After [in
 
 ## Get the network information
 
-Because only 200 participants can join testnet 3.2b, you'll first need to get the peer addresses of the seed nodes. The 200 participants have already been selected, but you can sign up for the waitlist [here](http://bit.ly/StakingSignup) in case a spot opens up. 
+Because only 400 participants can join testnet 3.3, you'll first need to get the peer addresses of the seed nodes. The 400 participants have already been selected, but you can sign up for the waitlist [here](http://bit.ly/StakingSignup) in case a spot opens up. 
```

Comment:
```suggestion
Because only 500 participants can join testnet 3.3, you'll first need to get the peer addresses of the seed nodes. The 500 participants have already been selected, but you can sign up for the waitlist [here](http://bit.ly/StakingSignup) in case a spot opens up. 
```